### DISCUSSION TITLE
Chap 7  P37 String unique  疑似判题错误？

### DISCUSSION THREAD

**Discussion by NingmengLemon** [datawhalechina/learn-python-the-smart-way-v2#26] [提问答疑] [answered]
疑似，只是疑似
[Chap 7  P37 String unique](https://hydro.ac/d/datawhale_p2s/p/37)
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/60469383/5ad6c066-3f40-4804-a0ff-8dae04c7e8ab)
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/60469383/5fbc6eee-52ec-4bda-aa98-e41ffa6fc2f0)
为了排除我函数写错的可能，我还用了两种方式来求解
但是感觉这个“应为……”的提示文本有些，，可疑？
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/60469383/4eb0648e-b1c1-42e0-8145-e3a84a42c384)
于是我尝试直接打印生成的列表，结果对了一个？
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/60469383/d68e4d82-bc7d-4324-9851-8ff55d338f7f)
再次为了排除我脑抽读题出错的可能，我特意关注了一下样例输出，**我个人觉得** 把字符串拼到一起之后再输出应该才是对的

**Comment by anine09** [comment]
抱歉 Chap 7 的作业仍在测试，我们不保证其正确性。

骆秀韬
聪明办法学 Python 教学团队

**Reply by anine09** [reply]
你可与 @zarjun 继续沟通这道题目。

**Comment by zarjun** [comment, marked as answer]
感谢您的热情提问，经过排查，这题确实存在判题错误❌，你的观点是正确的且受鼓舞的！
该问题已经尝试于2023年10月20日01点12分做出修复。在此为判题失误造成的学习体验影响表示抱歉
祝好！







在 2023-10-17 22:03:56，"柠檬味的凝萌" ***@***.***> 写道：

疑似，只是疑似
Chap 7 P37 String unique


为了排除我函数写错的可能，我还用了两种方式来求解
但是感觉这个“应为……”的提示文本有些，，可疑？

于是我尝试直接打印生成的列表，结果对了一个？

再次为了排除我脑抽读题出错的可能，我特意关注了一下样例输出，我个人觉得 把字符串拼到一起之后再输出应该才是对的
